### DISCUSSION TITLE
Feature Request: tasks to retrieve (and/or edit) the tester list for a given track

### DISCUSSION THREAD

**Discussion by erawhctim** [Triple-T/gradle-play-publisher#1146] [Ideas]
It'd be convenient if this plugin had a task to retrieve the list of tester email addresses for a given test track. Editing the list (to be able to add/remove testers) would be a nice addition as well, but retrieving the list covers most of my request.

This is somewhat adjacent to the core goal of this plugin (but not required functionality), so I could expect some hesitation to supporting this idea. Using the Play Developer API directly is painful, and using this plugin is nice and easy 🙂

**Comment by SUPERCILEX** [comment]
If there's an API to do so, I'd accept a PR.
